app/views/index/logs.phtml

```diff
@@ -14,7 +14,7 @@
 	?>
 
 	<?php if (!empty($items)) { ?>
-	<?php $this->logsPaginator->render('logs_pagination.phtml', 'page'); ?>
+	<?php $this->logsPaginator->render('logs_pagination.phtml', 0); ?>
```

Comment:
This was wrong, breaking the pagination. Fixed in https://github.com/FreshRSS/FreshRSS/pull/5403